project/Dependencies.scala

```diff
@@ -111,6 +111,14 @@ object Dependencies {
 
   private val dropwizardMetricsCore = "io.dropwizard.metrics" % "metrics-core" % "3.2.2" excludeAll (excludeSlf4j: _*)
 
+  private val jacksonFamily =
+    libraryFamily("com.fasterxml.jackson.core", JacksonVersion)(
+      "jackson-annotations", "jackson-core", "jackson-databind"
+    ) ++ libraryFamily("com.fasterxml.jackson.datatype", JacksonVersion)(
+      "jackson-datatype-jdk8", "jackson-datatype-jsr310", "jackson-datatype-guava", "jackson-datatype-pcollections"
+    )
+
```

Comment:
I just moved this up from lower down in the file to eliminate a warning from IDEA about "suspicious forward reference". I don't think it was actually a problem, but it's good to have this here with all of the other similar `val`s anyway.